show/reboot_cause.py

```diff
@@ -23,6 +24,100 @@ def read_reboot_cause_file():
 
     return reboot_cause_dict
 
+# Function to fetch reboot cause data from database
+def fetch_reboot_cause_from_db(module_name):
+    table = []
+    r = []
+    wrapper = textwrap.TextWrapper(width=30)
```

Comment:
There is no need to use TextWrapper. It is the responsibility of `tabulate` module to wrap the data based on the screen size.

Reply:
The new implementation doesn't have TextWrap